### ISSUE TITLE
Add Up1.ca as an alternative to img.bi

### ISSUE THREAD

**Issue by k3d3** [nylira/prism-break#1477]
Currently you show img.bi on prism-break's website, which I'm sure you know has been down for a few months now with no signs of coming back.

I wanted to recommend an alternative: Up1.

https://up1.ca/

https://github.com/Upload/Up1

It's somewhat recent - only been around for a year or so - however it's much better than img.bi in several ways:

1) Faster and more efficient - img.bi takes the raw image data, encrypts it, then encodes it into base64 and puts it into a JSON blob. These steps make it much slower than it needs to be due to the extra passes, and the base64 makes the uploaded file much bigger than the original. Up1 takes the raw image data, encrypts it, then uploads the encrypted blob the way it is. Additionally, they bring in all of AngularJS and then some, and it loads slower because of that.

2) img.bi's website was fairly cluttered, whereas Up1 basically has a big Upload button. On top of that, the URLs are half the size and are still very secure (128-bit security level).

3) Up1 also works as a pastebin, and can also host short video and audio files, or general files (any file that's less than 50MB).

4) ShareX support, which a ton of people seem to use.

TL;DR Up1 is better in many ways, and is just as secure (if not more secure) as img.bi.

(disclaimer, I'm one of the developers of Up1, and personally use it daily - feel free to ask questions if you have any!)

**Comment by Zegnat** [comment]
I might have a sit-down with the Up1 source this week, but it looks like a pretty fine replacement. img.bi’s own issue tracker hasn’t been kept up-to-date about the hosting situation either, so I would not feel bad swapping thes out.

**Comment by RoseTheFlower** [comment]
Riseup now use the same code to host their own version of this service:
https://share.riseup.net

**Comment by Zegnat** [comment]
It looks pretty good to me, and might also just replace [ProtectedText](https://prism-break.org/en/projects/protectedtext/) as an actual self-hostable alternative.

**Comment by Zegnat** [comment]
Closing this because we always prefer discussion to happen on pull requests instead. Please see #1491.
